lib/sidekiq/bulk.rb

```diff
@@ -8,6 +10,12 @@ def push_bulk(items, &block)
 
     Sidekiq::Client.push_bulk("class" => self, "args" => args)
   end
+
+  def push_bulk!(items, limit: 10_000, &block) # TODO: discuss this
```

Comment:
It's up for debate whether 10,000 is a reasonable limit here. When I was first experimenting with `push_bulk`, 10,000 seemed to keep time-to-enqueue-in-Redis reasonably low.